src/auto-evo/RunResults.cs

```diff
@@ -675,6 +686,20 @@ void OutputPopulationForPatch(Species species, Patch patch, long population)
 
                         builder.Append('\n');
                     }
+
+                    if (numberOfPatches < 2)
+                    {
+                        world.LogWorldEvent(new LocalizedString("TIMELINE_SPECIES_MIGRATED_TO",
+                                entry.Species.FormattedName, TranslationServer.Translate(
+                                    entry.SpreadToPatches[0].To.Name)), entry.Species.PlayerSpecies,
+                            "res://assets/textures/gui/bevel/newSpecies.png");
+                    }
+                    else
+                    {
+                        world.LogWorldEvent(new LocalizedString("TIMELINE_SPECIES_MIGRATED_TO_MULTIPLE",
```

Comment:
Wouldn't it be better to numerically show 2 of the migrations at least? That would have a higher chance of always showing the info, but won't take too much space.

Also the term "migration" perhaps needs tweaking as species can spread to a patch they are already in. Then again I suppose "spread to" would be even more misleading...

Reply:
Migrate in its broader sense can also mean relocate/move, in this case it's the species' sub-population to another place its kind already inhabits and not necessarily mean the entire species. Perhaps for this the wording could be "All or part of (species)'s population has migrated to (patch)"?

Reply:
All the population can't be part of a spread action in auto-evo, so that's irrelevant to include in the wording of the text.

Reply:
> Wouldn't it be better to numerically show 2 of the migrations at least? That would have a higher chance of always showing the info, but won't take too much space.

Btw, I never see species that migrated into multiple patches in a single auto-evo run so this may not even currently happen in the game at all.

Reply:
It doesn't currently (out of a few candidate migrations only the most beneficial is selected), but it wouldn't be very fair to keep AI species from doing that once the player can command their own population around.